src/calendar-list/index.js

```diff
@@ -59,22 +61,23 @@ class CalendarList extends Component {
     showScrollIndicator: false,
     scrollEnabled: true,
     scrollsToTop: false,
-    removeClippedSubviews: Platform.OS === 'android' ? false : true
+    removeClippedSubviews: Platform.OS === 'android' ? false : true,
+    scrollToDate: true
```

Comment:
What is 'scrollToDate' prop has to do with fixing borderRadius?

Reply:
I think this was a different issue where the calendar would scroll to the date automatically, even if you don't necessarily want it to.